chpl/chpl-service/src/main/java/gov/healthit/chpl/realworldtesting/manager/RealWorldTestingReportService.java

```diff
@@ -173,48 +174,58 @@ private String formatUserData(UserDTO user) {
     }
 
     private RealWorldTestingReport addMessages(RealWorldTestingReport report) {
-        if (isWithdrawn(report.getCurrentStatus())) {
+        if (isWithdrawn(report)) {
             report.setRwtPlansMessage(errorMsg.getMessage("realWorldTesting.report.listingWithdrawnMessage"));
-        } else if (isRwtPlansEmpty(report)) {
-            if (BooleanUtils.isTrue(report.getIcs())
-                    && (arePlansLateWarning(report.getRwtEligibilityYear()) || arePlansLateError(report.getRwtEligibilityYear()))) {
-                report.setRwtPlansMessage(errorMsg.getMessage("realWorldTesting.report.eligibleByIcs.missingPlansError",
-                        report.getRwtEligibilityYear().toString()));
-            } else if (arePlansLateWarning(report.getRwtEligibilityYear())) {
-                report.setRwtPlansMessage(errorMsg.getMessage("realWorldTesting.report.eligibleBySelf.missingPlansWarning",
-                        report.getRwtEligibilityYear().toString(),
-                        getPlansLateDate(report.getRwtEligibilityYear()).toString()));
-            } else if (arePlansLateError(report.getRwtEligibilityYear())) {
-                report.setRwtPlansMessage(errorMsg.getMessage("realWorldTesting.report.eligibleBySelf.missingPlansError",
-                        report.getRwtEligibilityYear().toString(),
-                        getPlansLateDate(report.getRwtEligibilityYear()).toString()));
-            }
+            report.setRwtResultsMessage("");
+        } else {
+            report.setRwtPlansMessage(getPlansMessage(report));
+            report.setRwtResultsMessage(getResultsMessage(report));
         }
-        if (isRwtResultsEmpty(report)) {
-            if (BooleanUtils.isTrue(report.getIcs())
-                    && (areResultsLateWarning(report.getRwtEligibilityYear()) || areResultsLateError(report.getRwtEligibilityYear()))) {
-                report.setRwtPlansMessage(errorMsg.getMessage("realWorldTesting.report.eligibleByIcs.missingResultsError",
-                        report.getRwtEligibilityYear().toString()));
+        return report;
+    }
+
+	private String getResultsMessage(RealWorldTestingReport report) {
+		if (isRwtResultsEmpty(report)) {
+            if (BooleanUtils.isTrue(report.getIcs()) 
+            		&& (areResultsLateWarning(report.getRwtEligibilityYear()) || areResultsLateError(report.getRwtEligibilityYear()))) {
```

Comment:
Did you see the checkstyle comment about the "tab" character? I don't really care as much about the trailing spaces ones, but definitely do not like tabs. Maybe there's something in your configuration that isn't respecting our checkstyle rules?